packages/backend/src/users/users.module.ts

```diff
@@ -1,6 +1,6 @@
 import { Module } from '@nestjs/common';
 import { UsersController } from './users.controller';
-import { DbModule } from '../db/db.module';
+import { DbModule } from 'src/db/db.module';
```

Comment:
\nit should change the import back to `import { DbModule } from '../db/db.module'`